test/include/psen_scan_v2/ros_integrationtest_helper.h

```diff
@@ -73,6 +73,27 @@ MATCHER_P(messageEQ, expected_msg, "")
   return expected_msg == *actual_msg;
 }
 
+// template <String topicName, int queueSize>
+template <typename T>
+class SubscriberMock
```

Comment:
Consider moving the class it to an own .h file. (Did I write `ros_integrationtest_helper.h`? Sorry, I did not think much about that)